module/epochs.go

```diff
@@ -36,3 +36,49 @@ type QCContractClient interface {
 	// cluster QC aggregator smart contract for the current epoch.
 	Voted(ctx context.Context) (bool, error)
 }
+
+// DKGContractClient enables interacting with the DKG smart contract. This
+// contract is deployed to the service account as part of a collection of
+// smart contracts that facilitate and manage epoch transitions.
+//
+// Phase parameters may be one of [1,2,3].
+//
+// TODO type for DKGPhase?
+// TODO type for DKGMessage?
+// TODO type for DKGResult? - should be all participant public keys plus group public key
+type DKGContractClient interface {
```

Comment:
I think each function needs an extra parameter for the Epoch ID, because messages and results should be grouped by Epoch.